src/OSPSuite.Infrastructure.Import/Services/ICsvSeparatorSelector.cs

```diff
@@ -1,10 +1,16 @@
 ﻿namespace OSPSuite.Infrastructure.Import.Services
 {
+   public class CSVSeparators
+   {
+      public char ColumnSeparator { get; set; }
+      public char DecimalSeparator { get; set; }
+   }
+
    public interface ICsvSeparatorSelector
    {
       /// <summary>
-      /// Returns the separator to be used for the reading of the .csv file
+      ///    Returns the separator to be used for the reading of the .csv file
       /// </summary>
-      char? GetCsvSeparator(string fileName);
+      CSVSeparators GetCsvSeparator(string fileName);
```

Comment:
feels like a good usage of named tuples here. (not required but simple API instead of having a class etc.)